translate/v23tov30/v23tov30.go

```diff
@@ -233,9 +233,17 @@ func translateGroups(groups []old.PasswdGroup) (ret []types.PasswdGroup) {
 
 func translateUnits(units []old.Unit) (ret []types.Unit) {
 	for _, u := range units {
+		var enabled *bool
+		// this follows Ignition v0.x logic except it let's Enabled=false win over Enable=true
```

Comment:
Any reason not to mention which logic?

The fact that Ignition accepts Enable=true over Enabled=false is subtle (since it depends on systemd's preset parsing) and is likely a bug.  I agree that the logic here is correct, but we should document it thoroughly.

Reply:
updated with a link :)